clusters/build-clusters/01_cluster/sshd-bastion/coredns-ma/deployment.yaml

```diff
@@ -0,0 +1,84 @@
+kind: List
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: bastion-coredns
+    name: coredns-ma
+    labels:
+      app: prow
+      component: coredns
+  spec:
+    selector:
+      app: prow
+      component: coredns
+    ports:
+    - name: dns-udp
+      port: 53
+      targetPort: dns-udp
+      protocol: UDP
+    - name: dns-tcp
+      port: 53
+      targetPort: dns-tcp
+      protocol: TCP
+    type: ClusterIP
+    clusterIP: 172.30.23.57
```

Comment:
Is this occupied by the current SVC?
In that case, remove this field and add it later after deleting the old SVC.
Otherwise this will cause failure on our post-submit and alert our triage.